### DISCUSSION TITLE
not working, unable to create URL links

### DISCUSSION THREAD

**Discussion by dozero-rlg** [microsoft/AzUrlShortener#285] [Q&A]
I first attempted a headless installation to integrate with a SharePoint list but kept receiving a "claim not accepted" error. I noticed the post from @rforster-dev but was confused by the postman documentation and couldn't make any correlation. I redeployed using the Blazor frontend but receive _"An error has occurred. This application may no longer respond until reloaded. Reload"_ when attempting to create a URL after logging in. The documentation doesn't appear to have been updated so i am not sure how to proceed to make this work.

**Comment by fboucher** [comment]
Thank you @dozero-rlg I notice some issues yesterday too.
I will investigate.
